src/styles/ingredients.scss

```diff
@@ -0,0 +1,182 @@
+.ingredients {
+  margin-bottom: 100px;
+
+  @include tablet {
+    position: relative;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 201px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__title {
+    position: relative;
+    max-width: 280px;
+    margin-bottom: 20px;
+
+    @include tablet {
+      margin-bottom: 310px;
+    }
+
+    @include desktop {
+      max-width: 380px;
+      margin-bottom: 287px;
+    }
+  }
+
+  &__blok {
+    display: block;
+    background: url(../images/svg/bank-mobile.svg) no-repeat right/contain;
+    width: 100%;
```

Comment:
You should add a fallback background color. This ensures that an appropriate color is displayed if the image is not available.